.github/workflows/ossar-analysis.yml

```diff
@@ -44,6 +56,6 @@ jobs:
 
       # Upload results to the Security tab
```

Comment:
here we still use `ossar-action` v1, while your fix is included in v2.
`ossar-action` config should be updated to:
```yml
- name: Run OSSAR
  uses: github/ossar-action@v2.0.0
  id: ossar
```

Reply:
Updated thanks - I apologize, I came late to realize about the versioning and so for the upload sarif's version. I can see just now that about the release's updated version.